gh-pages/action.yml

```diff
@@ -21,7 +25,7 @@ runs:
     - name: Setup Deno
       uses: denoland/setup-deno@v1
```

Comment:
Perhaps, `setup-deno` should be updated to v2, so it takes Deno 2 by default.

Reply:
Ah yes I missed this, thanks. Fixed.